### PR TITLE
javascript-methods/ | Keivan Mojmali

### PR DESCRIPTION
**Pull request by keivanmojmali**
image

### CODE REVIEW

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+};
+
+var warPeace = {
+  title:'War and Peace',
+  author:'Leo Tolstoy',
+};
+
+library.push(years);
+library.unshift(warPeace);
+library.splice(1);
```
Comment (Zhiwutian) [inline javascript-methods/main.js:59]: there is an issue here. The final console log should show two books in the library, and you are currently only showing one!

File: javascript-methods/main.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+console.log('Value of random index: ', randomIndex);
+
+var randomHero = heroes[randomIndex];
+
+console.log('Value of randomHero: ',randomHero);
+
+
+var library = [
+  book1 = {
```
Comment (Zhiwutian) [inline javascript-methods/main.js:25]: you should not have undeclared variables like this here. when you want to add objects into an array I would recommend the following: 

```javascript
var arrayWithObjects = [{},{},{}]
```

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple updates!

**Comment by keivanmojmali** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**